crates/i2c/src/i2c.rs

```diff
@@ -299,7 +306,7 @@ pub(crate) struct I2cReq {
 /// functionality without the need of a physical device.
 pub(crate) trait I2cDevice {
     // Open the device specified by the adapter number.
-    fn open(adapter_no: u32) -> Result<Self>
+    fn open(adapter_name: &str) -> Result<Self>
```

Comment:
Might it make sense to keep both? 

Anyway, the comment above should be update.